templates/books-show.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1">
+      <div class="panel panel-default">
+        <div class="panel-body">
+          <ul class="list-group form-inline">
+            <div class="row">
+              <div class="col-md-3 col-xs-12 thumbnail">
+                <img src="{{booksShowCtrl.book.image}}">
+              </div>
+              <div class="col-md-9 col-xs-12">
+                  <li class="list-group-item">
+                    <h4 class="inline-header">Book Title: {{booksShowCtrl.book.title}}</h4>
+                    <h4 class="inline-header">Author: {{booksShowCtrl.book.author}}</h4>
+                    <h4 class="inline-header">Date Released: {{booksShowCtrl.book.releaseDate}}</h4>
+                  </li>
+              </div>
+              <div class='panel-footer'>
```

Comment:
Why do you use single quotes sometimes, and double quotes other times? Consistency is good.